classes/game.rb

```diff
@@ -0,0 +1,30 @@
+class Game
+  attr_reader :multiplayers, :lastplayed_date
+  attr_accessor :title, :archived
+
+  def initialize(multiplayers, lastplayed_date)
+    @multiplayers = multiplayers
+    @lastplayed_date = lastplayed_date
+    @archived = false
```

Comment:
It would be great if you can use `super()` method to inherit all the props from the `Item` class, it should look something like this :
`def initialize(publish_date, archived, multiplayers, lastplayed_date)`
`super(publish_date, archived)`
    `@multiplayers = multiplayers`
    `@lastplayed_date = lastplayed_date`
`end`